rpcq/_base.py

```diff
@@ -173,8 +173,14 @@ def from_msgpack(b, *, max_bin_len=MAX_BIN_LEN, max_str_len=MAX_STR_LEN):
     #   Otherwise, unpack to Python str (or unicode on Python 2) by decoding with UTF-8 encoding (recommended).
     #   In msgpack >= 0.6, max_xxx_len is reduced from 2 GB to 1 MB, so we set the relevant ones
     #       to 2 GB as to not run into issues with the size of the values returned from rpcq
-    return msgpack.loads(b, object_hook=_object_hook, raw=False,
-                         max_bin_len=max_bin_len, max_str_len=max_str_len)
+    return msgpack.loads(
+        b,
+        object_hook=_object_hook,
+        raw=False,
+        max_bin_len=max_bin_len,
+        max_str_len=max_str_len,
+        strict_map_key=False,
```

Comment:
Note support in `0.6.2`: https://github.com/msgpack/msgpack-python/blob/997b524f06176aaa6bd255a046a8746e99b4f87d/msgpack/fallback.py#L182